### DISCUSSION TITLE
[SQL Server] Generating DDL duplicating Unique Constraint

### DISCUSSION THREAD

**Discussion by gabrielrra** [dbeaver/dbeaver#15049] [Q&A]
#### System information: 
- Windows 10
- DBeaver version - 21.3.0.202111281534
- No extensions

#### Connection specification:
- Sql Server 2005 and 2019

#### Describe the problem you're observing:

Whenever I generate a DDL of a table with a Unique Constraint, the sql resulted will have two lines for creating the Unique Constraint, one inside the `CREATE TABLE ( )` and another one separated as an `CREATE UNIQUE NONCLUSTERED INDEX ` statement, as seen below

![image](https://user-images.githubusercontent.com/39421688/144240610-4708553e-220b-412f-b0a9-c4a1cbc8127c.png)
No matter what checkbox I check under the **Settings** part, it always duplicates.

This leads to an error when running the whole SQL because the constraint already exists when the `CREATE UNIQUE NONCLUSTERED INDEX` is reached.

#### Steps to reproduce, if exist:
Create a table with at least one Unique Constraint and then use Dbeaver to automatically generate DDL sql.

**Comment by LonwoLonwo** [comment]
Hello @gabrielrra 

As SQL Server [documentation](https://docs.microsoft.com/en-us/sql/relational-databases/tables/unique-constraints-and-check-constraints?view=sql-server-ver15) said:

> The Database Engine automatically creates a UNIQUE index to enforce the uniqueness requirement of the UNIQUE constraint.

 So... we can't show you your table index. It is correct. 
So what do you suggest?
